src/library_browser.js

```diff
@@ -901,7 +905,9 @@ var LibraryBrowser = {
    * @param {boolean=} noSetTiming
    */`,
   $setMainLoop: (browserIterationFunc, fps, simulateInfiniteLoop, arg, noSetTiming) => {
+#if ASSERTIONS
     assert(!Browser.mainLoop.func, 'emscripten_set_main_loop: there can only be one main loop function at once: call emscripten_cancel_main_loop to cancel the previous one before setting a new one with different parameters.');
```

Comment:
@sbc100 I'm not sure how to test this. I have an open-source pet project that's failing to build because assert is undefined here, and I'm having issues with using emscripten from source that's preventing me from testing locally

Reply:
The test you added looks like it should cover this case, no?   Or it just that you can't verify 100% that that is true?

I'm happy to help you get setup for local testing if that would help.

The simplest thing to do is:

1. Install and setup emsdk as normal.
2. `export EM_CONFIG=/path/to/emsdk/.emscripten`
3. Go to your emscripten checkout (anywhere you have it)
4. Run `./test/runner browser.test_xxx`